docs/guide/repository.md

```diff
@@ -602,29 +495,19 @@ This cache backend is using [Redis, a in-memory data structure store](http://re
 **Example**
 
 ``` yaml
-# config.yml example
-stash:
-    caches:
-        default:
-            drivers: [ Redis ]
-            Redis:
-                servers:
-                    -
-                        server: 'redis1.ez.no'
-                        port: 6379
-                    -
-                        server: 'redis2.ez.no'
-                        port: 6379
+services:
+    cache.redis:
+        parent: cache.adapter.redis
+        tags:
+            - name: cache.pool
+              clearer: cache.app_clearer
+              provider: 'redis://secret@example.com:1234/13'
```

Comment:
we'll also need to tell people to reconfigure repository `cache_service_name` setting.